lasagne/layers/special.py

```diff
@@ -815,8 +864,8 @@ def _initialize_tps(num_control_points):
         for point_2 in range(point_1, num_control_points):
 
             L[point_1 + 3, point_2 + 3] = _U_func_numpy(
-                    source_points[0, point_1], source_points[1, point_1],
-                    source_points[0, point_2], source_points[1, point_2])
+                source_points[0, point_1], source_points[1, point_1],
+                source_points[0, point_2], source_points[1, point_2])
```

Comment:
I'd prefer double indent as before (make sure to use the same pep8 version as in `requirements-dev.txt`).